core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerUtils.scala

```diff
@@ -17,15 +17,18 @@
 
 package org.apache.spark.scheduler.cluster.mesos
 
-import java.util.List
 import java.util.concurrent.CountDownLatch
+import java.util.{List => JList}
```

Comment:
also here, class imports go before more deeply nested packages (its not just lexicographic), so

```
import java.util.{List => JList}
import java.util.concurrent.CountDownLatch
```